Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "cgl"
 license = "MIT / Apache-2.0"
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
This should be 0.2, since we use types from gleam in the publically-exported API in cgl-rs.